lib/pos_consensus.go

```diff
@@ -719,12 +719,14 @@ func (fc *FastHotStuffConsensus) produceUnsignedBlockForBlockProposalEvent(
 	}
 
 	// Build a UtxoView at the parent block
-	utxoViewAtParent, err := fc.blockchain.getUtxoViewAtBlockHash(*parentBlockHash)
+	utxoViewAtParentAndUtxoOps, err := fc.blockchain.getUtxoViewAtBlockHash(*parentBlockHash)
```

Comment:
nit: naming consistency
```suggestion
	parentUtxoViewAndUtxoOps, err := fc.blockchain.getUtxoViewAtBlockHash(*parentBlockHash)
```